docs/build/configure/_index.md

```diff
@@ -134,29 +139,15 @@ See [Example JSON configuration file](/internals/local-configuration-file/#examp
 
 {{% /expand %}}
 
-The **CONFIGURE** tab has subtabs for each section of your machine's config:
```

Comment:
Can you create a follow up ticket to review especially this page after R2D2 is fully done? I'm concerned about losing this overview - though I like that the headlines are now more task-oriented

Reply:
Sounds good, will do!